### PR TITLE
feat(zplug): print separator for zplug step

### PR DESCRIPTION
**Pull request by ericmarkmartin**

### CODE REVIEW

File: src/steps/os/unix.rs [hunk 1]
```diff
@@ -9,6 +9,7 @@ use std::process::Command;
 
 pub fn run_zplug(base_dirs: &BaseDirs, run_type: RunType) -> Result<(), Error> {
     let zsh = require("zsh")?;
+    print_separator("zplug");
```
Comment (r-darwish) [inline src/steps/os/unix.rs]: Oops. I probably forgot that.

Anyway, separators should be printed after all requirements have been made. The current implementation will print the zplug separator even if you only have zsh install and don't use zplug. You should move that line before the cmd line.

Comment (r-darwish) [reply src/steps/os/unix.rs]: You still need to remove this one

**Comment by ericmarkmartin** [comment]
Sorry about that. That's what you get for late night coding, I suppose. Should be fixed now.

**Pull request merged**